presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java

```diff
@@ -186,6 +187,22 @@ public void renameTable(String databaseName, String tableName, String newDatabas
         alterTable(databaseName, tableName, table);
     }
 
+    @Override
+    public void commentTable(String databaseName, String tableName, Optional<String> comment)
+    {
+        Optional<org.apache.hadoop.hive.metastore.api.Table> source = delegate.getTable(databaseName, tableName);
+        if (!source.isPresent()) {
+            throw new TableNotFoundException(new SchemaTableName(databaseName, tableName));
+        }
+        org.apache.hadoop.hive.metastore.api.Table table = source.get();
+
+        Map<String, String> parameters = table.getParameters();
+        parameters.put(TABLE_COMMENT, comment.orElse(""));
```

Comment:
If `comment` is `Optional.empty()`, the comment should be removed. Does the hive metastore support removing comments explicitly or does it consider an empty comment the same as not having a comment? What happens if after clearing the comment you run `DESCRIBE <table>`? Does it show `COMMENT ''`?

Reply:
hive supports both empty comment and removing a comment (`ALTER UNSET`). Actually, I tryied to remove the comment when it is empty, but followed the hive behaviour. Which do you think is better?
```sql
hive> ALTER TABLE table1 SET TBLPROPERTIES ('comment' = '');
hive> desc formatted table1;
...
Table Parameters:	 	 
	comment             	                    
	last_modified_by    	root                
...
```

```sql
hive> ALTER TABLE table1 UNSET TBLPROPERTIES ('comment');
hive> desc formatted table1;
...
Table Parameters:	 	 
	last_modified_by    	root                
...
```